source/legacy.txt

```diff
@@ -37,6 +40,26 @@ In addition to updating your application to handle any necessary changes,
 always check for any other differences in options and return values before
 moving it to production.
 
+Current API Benefits
+--------------------
+
+You can continue to use the legacy API with each new MongoDB server release. 
+However, the legacy API does not support any new features introduced in MongoDB 3.0 
+or later. Upgrade to the current API to use the following features:
+
+- **Change streams**, a MongoDB Server feature designed to monitor real-time changes to a single 
+  collection, database, or deployment
+- **Multi-document ACID transactions**, which ensure atomicity of reads and writes to multiple 
+  documents and allow for transactions across multiple operations, collections, databases, 
+  documents, and shards
+- **Time series collections**, or collections that store sequences of measurements over a period 
```

Comment:
S: remove "or" to be consistent with other list items

```suggestion
- **Time series collections**, collections that store sequences of measurements over a period 
```

Reply:
Updated, I used slightly different wording than your suggestion but made it more consistent with the other bullet points